graphene_django/filter/tests/test_fields.py

```diff
@@ -608,6 +608,7 @@ def resolve_all_reporters(self, info, **args):
     assert result.data == expected
 
 
+@pytest.mark.skip(reason="no longer relevant?")
 def test_should_query_filter_node_double_limit_raises():
```

Comment:
I think you can remove this test @zbyte64

Reply:
Done.